docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+- Answer questions about the protocol, services and products.
+
+
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
+
+After discussion, if the community agrees that the change should be implemented, the proposer may also make a Treasury proposal to support the work. Check out the treasury proposals section.
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way. 
+
+In addition you should:
+
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT). If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance. See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
+
+Governance approval for a Treasury spend will also  includes list of the deliverables as outlined in the proposal.
+
+## Bug Reports
+
+We try our best, but bugs are an everyday reality with all software and are bound to happen. We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.
+
+If you are unsure that a bug is a bug, it is best to open an issue and report it anyway. The active developers will evaluate it and help to figure out the issue.
+
+It is helpful to check if a report has already been filed in the related project. Search the issues board for possible phrases that match the description of the bug. It's possible you may not find an issue, but it's better to file a duplicated bug than not report one.
+
+Once you begin reporting the bug, write a descriptive title so that if others find the same issue they can either add to your findings or know that the bug has already been reported. A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets. The more details you provide, the easier it is to fix the issue.
```

Comment:
```suggestion
We try our best, but bugs are an everyday reality with all software and are bound to happen.
We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.

If you are unsure that a bug is a bug, it is best to open an issue and report it anyway.
The active developers will evaluate it and help to figure out the issue.

It is helpful to check if a report has already been filed in the related project.
Search the issues board for possible phrases that match the description of the bug.
It's possible you may not find an issue, but it's better to file a duplicated bug than not report one.

Once you begin reporting the bug, write a descriptive title so that if others find the same issue they can either add to your findings or know that the bug has already been reported.
A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets.
The more details you provide, the easier it is to fix the issue.
```